operator/builtin/output/count/count.go

```diff
@@ -0,0 +1,149 @@
+package counter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math"
+	"math/big"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/observiq/stanza/entry"
+	"github.com/observiq/stanza/operator"
+	"github.com/observiq/stanza/operator/helper"
+)
+
+type CountOutputConfig struct {
+	helper.OutputConfig `yaml:",inline"`
+	Path                string          `json:"path,omitempty" yaml:"path"`
+	Duration            helper.Duration `json:"duration,omitempty" yaml:"duration,omitempty"`
+}
+
+var defaultCounterDuration = helper.NewDuration(1 * time.Minute)
+
+func init() {
+	operator.Register("count_output", func() operator.Builder { return NewCounterOutputConfig("") })
+}
+
+func NewCounterOutputConfig(operatorID string) *CountOutputConfig {
+	return &CountOutputConfig{
+		OutputConfig: helper.NewOutputConfig(operatorID, "count_output"),
+		Duration:     defaultCounterDuration,
+	}
+}
+
+func (c CountOutputConfig) Build(bc operator.BuildContext) ([]operator.Operator, error) {
+	outputOperator, err := c.OutputConfig.Build(bc)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	logChan := make(chan struct{}, 1)
```

Comment:
What is the purpose of `logChan`? Doesn't look like it's used at all in this code except to listen on it. It's unexported so nothing can push down it.

Reply:
My idea was mostly effort of making tests less reliant on a `time.Sleep()` by throwing a message to the channel to invoke the message, but yeah its unexported and not used in any other capacity so I'll just remove it 👍